compute_tools/src/compute.rs

```diff
@@ -905,8 +905,21 @@ impl ComputeNode {
                 .query_pairs_mut()
                 .append_pair("application_name", "migrations");
 
-            let mut client = Client::connect(connstr.as_str(), NoTls)?;
-            handle_migrations(&mut client).context("apply_config handle_migrations")
+            match Client::connect(connstr.as_str(), NoTls) {
```

Comment:
Just a reminder to do not forget to resolve the conflict in favor of
```rust
            conf.application_name("compute_ctl:migrations");

            let mut client = conf.connect(NoTls)?;
```
or something